Dockerfile

```diff
@@ -11,6 +11,13 @@ ENV HOMEBREW_NO_AUTO_UPDATE=1
 # hadolint ignore=DL3002
 USER root
 
+RUN if [ "x${img}" = "xhomebrew/debian7" ]; then \
+      apt-get update \
+      && apt-get install -y --no-install-recommends gcc-4.9-backport=4.9.2-10+deb7u1 \
```

Comment:
Thought: maybe we should do this on our global wheezy image?

Reply:
Could do, though I think it's also a good idea to scope this to only what needs it, which is only portable-libxcrypt.

Reply:
cool 👍🏻